vllm/config.py

```diff
@@ -262,6 +263,14 @@ def __init__(
 
         self.maybe_pull_model_tokenizer_for_s3(model, tokenizer)
 
+        if (backend := envs.VLLM_ATTENTION_BACKEND
+            ) and backend == "FLASHINFER" and find_spec("flashinfer") is None:
+            raise ValueError(
+                "VLLM_ATTENTION_BACKEND is set to FLASHINFER, but flashinfer "
+                "module was not found."
+                "See https://github.com/vllm-project/vllm/blob/main/Dockerfile"
```

Comment:
It would be nice to have a proper page in the docs to document this

Reply:
Agree, will look into it

Reply:
@mgoin I've added a small section about backends because I wasn't sure where I could start mentioning flashinfer in the existing docs without proper context. Let me know what you think.